src/components/Footer.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import Link from 'next/link';
+import Image from 'next/image';
+import MouseEvent from 'react';
+import Logo from '@/images/logo-lg.svg';
+import CurveLine from '@/images/footer-curve-line.svg';
+
+export function Footer() {
+
+  const onContextMenuHandler = (e: MouseEvent) => {
+    if(e.preventDefault != undefined)
+      e.preventDefault();
+    if(e.stopPropagation != undefined)
+      e.stopPropagation();
+    return false;
```

Comment:
[Suggested] 有了 TypeScript 這邊的檢查就不需要了，然後感覺只需要 e.preventDefault() ?
```suggestion
      e.preventDefault();
```

Reply:
> "這邊的檢查就不需要了"

妳指的是 對於 **e.preventDefault != undefined** 和 **e.stopPropagation != undefined**
這樣的檢查嗎？

Reply:
不用判斷是不是 undefined，因為已經加上 type

Reply:
@CoolizzLuo, @ba1679 
➜ 已在 2024/6/02 08:58 的 commit : 56da986 移除此判斷 : "**undefined check**"